pr-deploy/src/github.ts

```diff
@@ -32,12 +32,12 @@ const owner = process.env.GH_OWNER;
 const repo = process.env.GH_REPO;
 
 export class PullRequest {
+  public headSha: string;
   private github: Octokit;
-  private headSha: string;
 
   constructor(github: Octokit, headSha: string) {
-    this.github = github;
     this.headSha = headSha;
+    this.github = github;
```

Comment:
Curious: does this order matter?

Reply:
It was just to make it consistent with above, where I switched the order since public should come before private properties.